docs/01-readme.md

```diff
@@ -52,7 +52,26 @@ make package
 sudo apt install *.deb
 ```
 
-
+#### NixOS (overlay in configuration.nix)
+```nix
+# Overlay bellow will build AwesomeWM from selected commit
+# NOTE: Manually change revision (commit numbers) on every update.
+
+nixpkgs.overlays = [ 
+(self: super: {
+  awesome = super.awesome.overrideAttrs (old: {
+    src = super.fetchFromGitHub {
+      owner = "awesomewm";
+      repo = "awesome";
+      rev = "4319b161103a81403ba3516924e86e13ff33c036";
+      sha256 = "1p7c3cmdy0iyk28fgc579rkzfhnwn1i1b541rs2nw80h1xzs1g80";
+        };
+    });
+  })
+];
+
+services.xserver.windowManager.awesome.enable = true;
+```
```

Comment:
nit, can you leave en empty line between lines 74 and 75. Doesn't affect the markdown compilation, but having this line would make the code more readable (imho) in raw text format